Makefile

```diff
@@ -75,9 +75,16 @@ build-unit-tests-image:
 run-unit-tests:
 	$(run_unit_tests_image) make test
 
+KUBERNETES_VERSION=1.23.1
```

Comment:
is this mandatory? in which recurring ticket do we need to bump this?

Reply:
I didn't want to use latest, do you think using latest will be better?

Reply:
no, `latest` is the worst. is it the default value?
if we must maintain it, we should at least have a recurring ticket for it

Reply:
ok so I will talk to Eran and he will add in the description of creating new Kubernetes tickets to add to edit this line, is it ok?

Reply:
depends - [is this env var mandatory](https://github.com/IBM/ibm-block-csi-operator/pull/248#discussion_r833302968)? is the default value `latest`?
i.e. which value will be used without this env var?

Reply:
the default is latest, so if you don't want to use default so yes this env var in mandatory

Reply:
so please ensure this new bump is contained in a recurring ticket

Reply:
ok I will talk to Eran about it